src/api/spec/controllers/webui/request_controller_spec.rb

```diff
@@ -394,6 +396,25 @@
     end
   end
 
+  describe 'POST #set_bugowner_request' do
+    context 'with valid parameters' do
+      before do
+        login(submitter_with_group)
+        post :set_bugowner_request, params: {
+          project: source_project_fluffy.name,
+          user: submitter_with_group.login,
+          group: submitter_with_group.groups.first.title,
+          description: 'blah blah blash'
+        }
+        @bs_request = BsRequest.where(creator: submitter_with_group.login, state: 'new').first
```

Comment:
what about `find` instead of `ẁhere` + `first`? what about adding this to a `let`?

Reply:
you mean ```find_by```?

Reply:
to have a ```find(:first, conditions:{})``` as fair as i know is deprecated

Reply:
I mean `find_by`